pages/Category.vue

```diff
@@ -1,125 +1,192 @@
 <template>
   <div id="category">
-    <header class="bg-cl-secondary py35 pl20">
-      <div class="container">
-        <breadcrumbs />
-        <div class="row middle-sm">
-          <h1 class="col-sm-8 category-title mb10">
-            {{ getCurrentCategory.name }}
-          </h1>
-          <div class="sorting col-sm-2 align-right mt50">
-            <label class="mr10">{{ $t("Columns") }}:</label>
-            <columns @change-column="columnChange" />
-          </div>
-          <div class="sorting col-sm-2 align-right mt50">
-            <sort-by
-              :has-label="true"
-              :value="getCurrentSearchQuery.sort"
-              @change="changeFilter"
-            />
-          </div>
-        </div>
+    <SfBreadcrumbs class="breadcrumbs desktop-only" :breadcrumbs="breadcrumbs">
+      <template #link="{breadcrumb}">
+        <router-link :to="breadcrumb.route.link">
+          {{ breadcrumb.text }}
+        </router-link>
+      </template>
+    </SfBreadcrumbs>
+    <div class="navbar section">
+      <div class="navbar__aside desktop-only">
+        <h1 class="navbar__title">Categories</h1>
```

Comment:
This string should be wrapped in translate function `$t('')` and added to default (en_US) translations if not exist there.

Reply:
Done.